ext/src/inject/inject.js

```diff
@@ -20,65 +22,60 @@ chrome.extension.sendMessage({}, function (response) {
 			configSetBasePath.onclick = onLinkClick;
 			lastGroup.after(configSetBasePath);
 			clearInterval(readyStateCheckInterval);
-
-			// ----------------------------------------------------------
-			// This part of the script triggers when page is done loading
-			console.log("Hello. This message was sent from scripts/inject.js");
-			// ----------------------------------------------------------
-
 		}
 	}, 10);
 });
 
 function getRepo(uri) {
-	const regex = /(?<repo>^https.+)\/pull-requests/gm;
-	const matches = regex.exec(uri);
+	const matches = regexRepo.exec(uri);
 	return matches.groups.repo;
 }
 
+function parseFileAndLine(uri) {
+	regexLine.lastIndex = 0;
```

Comment:
that's a bit of a hack. isn't there a cleaner way to make sure we start with a clean slate ?